TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/claw/Claw.java

```diff
@@ -0,0 +1,46 @@
+package org.firstinspires.ftc.teamcode.subsystems.claw;
+
+import com.arcrobotics.ftclib.command.SubsystemBase;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+public class Claw extends SubsystemBase {
+
+    private Finger rightFinger;
+    private Finger leftFinger;
+
+    public Claw(HardwareMap hardwareMap) {
+        this.rightFinger = new Finger(hardwareMap.servo.get(ClawConstants.RIGHT_SERVO_ID), ClawConstants.RIGHT_GRIPPER_OPEN_POSITION, ClawConstants.RIGHT_GRIPPER_CLOSE_POSITION);
+        this.leftFinger = new Finger(hardwareMap.servo.get(ClawConstants.LEFT_SERVO_ID), ClawConstants.LEFT_GRIPPER_OPEN_POSITION, ClawConstants.LEFT_GRIPPER_CLOSE_POSITION);
+    }
+
+    protected void openBothFingers() {
+        rightFinger.open();
+        leftFinger.open();
+    }
+
+    protected void closeBothFingers() {
+        rightFinger.close();
+        leftFinger.close();
+    }
+
+    protected void toggleRightFinger() {
+        rightFinger.toggleClaw();
+    }
+
+    protected void toggleLeftFinger() {
+        leftFinger.toggleClaw();
+    }
+
+    protected void switchFingers() {
+        Finger switchFinger = new Finger(rightFinger);
```

Comment:
dont use copy constructur

Reply:
Fun fact, every year _FIRST_ posts statistics about how teams are using WPILIB. During Rapid React, there was a team the created over 20 instances of a falcon, and some more instances of other motors (which is more motor instances than motors on the robot).
Don't use a copy constructor when you want to use the same instance.

Reply:
YOU ARE:
![1200 x 600](https://github.com/GreenBlitz/FTC-CENTERSTAGE/assets/108526263/a26d236c-de22-4b32-94e3-d9de1040c9da)